src/fu-util-common.c

```diff
@@ -1610,9 +1611,21 @@ fu_util_security_attrs_to_string (GPtrArray *attrs)
 				FwupdSecurityAttr *attr = g_ptr_array_index (attrs, i);
 				if (!fwupd_security_attr_has_flag (attr, hpi_suffixes[j]))
 					continue;
+				if (!help &&
+				    fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_RUNTIME_ISSUE) &&
+				    !fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS))
```

Comment:
woah, that's quite a high bar -- if you fail a HSI:5 check and you drop down to HSI:4 that's really still very high and I don't think we should show the notification. I'd much rather have two pages, one for runtime issues, and one if the HSI number is < 2 or something.

Reply:
OK i'll rename that page and make it runtime focused (as that's what the PR is).